prodigy.el

```diff
@@ -1279,7 +1287,7 @@ SIGNINT signal."
   "Jump to dired mode for service at point."
   (interactive)
   (-when-let (service (prodigy-service-at-pos))
-    (dired (prodigy-service-cwd service))))
+    (funcall prodigy-file-manager (prodigy-service-cwd service))))
```

Comment:
You should also update the name of this (`prodigy-jump-dired `) method and update the docs for it. There is also a binding to this function.

Reply:
Makes sense, I renamed that function and updated the doc, would it be ok what I wrote?